textx/scoping/providers.py

```diff
@@ -153,6 +162,18 @@ def _find_obj_fqn(p, fqn_name, cls):
             """
 
             def find_obj(parent, name):
+                if parent is not current_obj and \
```

Comment:
Do we need `current_obj` parameter in all these function. As I see it carries the value of top level `obj` and is never changed. Could we just say here `if parent is not obj and ...`  and remove all those `current_obj` params?

Reply:
Absolutely! I removed the superfluous parameters. However I changed the top-level parameter name to "current_obj" to make the code more readable.